tests/test-cases/Group1-Docker-Commands/1-02-Docker-Pull.robot

```diff
@@ -19,6 +19,85 @@ Suite Setup  Conditional Install VIC Appliance To Test Server
 Suite Teardown  Cleanup VIC Appliance On Test Server
 Test Timeout  20 minutes
 
+*** Keywords ***
+Cleanup MITM Test Detritus
+    Run Keyword  Destroy Proxified VCH
```

Comment:
that doesn't really do anything, don't need both Destroy Proxified VCH as well as this keyword...

Reply:
ahahaha yeah I had that thought while I was typing it out.. why does `Run Keyword` even exist as a keyword if it's never necessary?

Reply:
to be able to run a keyword as a variable,
i.e. Run Keyword  ${keyword1}  or ${keyword2}